src/argument_parser.rs

```diff
@@ -22,3 +27,34 @@ pub fn get_search_targets(args: &[String]) -> Vec<Cow<str>> {
         .filter_map(|arg| arg.strip_prefix("-T").map(Cow::Borrowed))
         .collect()
 }
+
+/// Exapnds @file arguments into the file's contents
+pub fn expand_files(args: &[String]) -> Vec<String> {
+    let mut expanded = Vec::with_capacity(args.len());
+
+    for arg in args {
+        if let Some(arg) = arg.strip_prefix('@') {
+            // The normal linker was able to open the file, so this *should* never panic
+            let file = File::open(arg).expect(&format!(
+                "Unable to open {arg}, this should never happen and should be reported"
+            ));
+            let reader = BufReader::new(file);
+            for line in reader.lines() {
+                // Same as above, normal linker succeeded so we should too
+                let line = line.expect(&format!(
+                    "Invalid file ({arg}), this should never happen and should be reported"
```

Comment:
Shouldn't that be?
```suggestion
                    "Invalid file (arg}, this should never happen and should be reported"
```

Reply:
What do you mean? Do you think it should be "{arg}", without the parentheses?

Reply:
Yes